tests/unit/dojson/test_dojson_hep.py

```diff
@@ -116,6 +116,77 @@ def test_external_system_numbers(marcxml_to_json, json_to_marc):
     assert (marcxml_to_json['external_system_numbers'][0]['obsolete'] ==
             json_to_marc['035'][0]['z'])
 
+def test_simple_language():
+    snippet = (
+        '<datafield tag="041" ind1=" " ind2=" ">'
+        '  <subfield code="a">Italian</subfield>'
+        '</datafield>'
+    )
+
+    expected = ["it"]
+    result = clean_record(hep.do(create_record(snippet)))
+
+    assert expected == result['languages']
```

Comment:
I just had an epiphany: it's easy to write tests that ensure that the rule satisfies the schema! You just need to fetch the schema, drill down to the right point and then call `validate` from `jsonschema` on that subschema. Nothing to fix here, I just wanted to write it down so I don't forget it.